tools/regression_test.sh

```diff
@@ -423,12 +430,18 @@ function run_local {
 }
 
 function setup_options_file {
-  if ! [ -z "$OPTIONS_FILE" ]; then
+  async_io="$1"
+  file=$OPTIONS_FILE
+  if [ "$async_io" == "true" ]; then
+    file=$OPTIONS_FILE_ASYNCIO
```

Comment:
Can we take async IO specific parameters as extra arguments and auto generate an OPTIONS file based on that? I think its less confusing to have the invoker of this script specify just one OPTIONS file that controls the database layout.

Reply:
I can do that as well. Similar to seek_next for async_io.